data/xcspec_evals.bzl

```diff
@@ -525,7 +525,7 @@ def _com_apple_compilers_llvm_clang_1_0__CLANG_ENABLE_MODULE_IMPLEMENTATION_OF__
             (eval_val_1_used_user_content, eval_val_1) = XCSPEC_EVALS[opt["DefaultValue"]](xcconfigs, id_configs)
             used_user_content = used_user_content or eval_val_1_used_user_content
 
-    return (used_user_content, (eval_val_0 and eval_val_1))
+    return (used_user_content, (eval_val_0 == "YES" and eval_val_1 == "YES"))
```

Comment:
Something to explicitly flag here is that this is intended to be codegen only from the `Xcode.app/Contents/PlugIns/XCBSpecifications.ideplugin/Contents/Resources/CoreBuildSystem.xcspec` file. Making manual tweaks in here likely will probably lead to surprises in the future. A proper fix likely should be somewhere else, either in the xcspec codegen code, in cocoapods-bazel handling, or somewhere else. This is potentially a reasonable stopgap, though those tend to stick around longer than anticipated.

Oddly, I don't see a diff in the `CLANG_ENABLE_MODULES` in that xcspec between 13.3.1 and 14.2.0.

Reply:
Definitely, I want to follow up on this but currently blocked from using `main` rules_ios due to this issue. This resolves it for now and it does just seem like some simple change we need to make in the code generator but I have 0 experience there so not sure if we want to wait for me/someone else to pick that up.